### ISSUE TITLE
Better (less scary, more informative) error pages

### ISSUE THREAD

**Issue by mialondon** [LibCrowds/libcrowds#553]
Huge error codes are pretty startling; a better user experience would reassure people that they didn't do anything wrong, ideally give them a human-readable explanation of what happened and what they should do next.

**Comment by alexandermendes** [comment]
Agreed that they could be a bit nicer. Just not something I could spend too many hours on at the time! Any examples of favourite error pages from other websites?

**Comment by alexandermendes** [comment]
See https://github.com/Scifabric/pybossa/releases/tag/v2.9.3 - the 415 errors when multiple users are going through a static IP will be fixed. Other errors have been updated a bunch over the months but we can improve them.

**Comment by alexandermendes** [comment]
If there's a quick fix to be made here to cover all of the main HTTP errors (400, 403, 404, 500 etc.) then let's do it. Otherwise, let's close this. Examples of error pages that you like from other sites would be good, although I'm probably not going to get into anything too design-heavy at this stage.

**Comment by mialondon** [comment]
@alexandermendes I haven't been able to hit an error page to check the text, but my last impression was that the technical text and error number startled people, so it'd be good to check what's there now first. Any tips for generating an error?

**Comment by alexandermendes** [comment]
Well hopefully we are at a stage where people aren't hitting many errors. 404s will probably be one of the more common ones though:

https://www.libcrowds.com/idontexist

The rest follow the same basic structure. They are pretty uninteresting and I'm up for updating them a bit but would be keen to keep things relatively simple.

**Comment by mialondon** [comment]
@alexandermendes I get weird errors about losing contact with the server sometimes - I'll keep an eye out for them

**Comment by mialondon** [comment]
@alexandermendes Got a 403 today. 
![20180713 libcrowds i li_ https___www libcrowds com_collection_playbills_projects_126_iiif-annotation](https://user-images.githubusercontent.com/380763/42690936-20546058-869e-11e8-9588-6c5954d29350.png)

Some generic headline text along the lines of 'Something went wrong. But don't worry, reload the page and it'll probably be fine' that's more prominent than the error message would be great.

**Comment by alexandermendes** [comment]
Although that actually only applies to those 403 errors on the task presenter page. For most other 4xx, 5xx errors refreshing the page won't help! For example, 404 if the page doesn't exist or 500 if there is some server error.

The above one, where there was a problem with the task submission, does seem to be the one we're generally referring to with these scary error messages. So, maybe we have something like the above "try reloading the page" suggestion whenever the task submission fails.
